packages/flutter_webrtc/example/flutter_webrtc_demo/tizen/tizen-manifest.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest package="org.tizen.flutter_webrtc_tizen_example" version="1.2.0" api-version="5.5" xmlns="http://tizen.org/ns/packages">
```

Comment:
I just run demo app on Tizen 6.0 ,6.5 and 7.0 on TV device. I'm not sure it support tizen 5.5.

Reply:
You didn't use any Tizen 6.0 API but just haven't test on Tizen 4.0 or 5.5 devices, right?

Reply:
```sh
$ objdump -T tizen/lib/armel/libwebrtc.so | ack GLIBC_2.2
00000000      DF *UND*  00000000  GLIBC_2.29  log2
00000000      DF *UND*  00000000  GLIBC_2.29  pow
00000000      DF *UND*  00000000  GLIBC_2.28  fcntl64
00000000      DF *UND*  00000000  GLIBC_2.29  exp
00000000      DF *UND*  00000000  GLIBC_2.27  powf
00000000      DF *UND*  00000000  GLIBC_2.27  expf
00000000      DF *UND*  00000000  GLIBC_2.29  log
00000000      DF *UND*  00000000  GLIBC_2.27  exp2f
00000000      DF *UND*  00000000  GLIBC_2.27  log2f
00000000      DF *UND*  00000000  GLIBC_2.27  logf
```
The `libwebrtc.so` library requires GLIBC_2.29 or above and thus is not compatible with Tizen 5.5 or older devices. So it is correct that the minimum required API version of this package is 6.0 (unless you rebuild the libs for Tizen 5.5). You can change the `api-version` value to 6.0.

Reply:
Sorry, my mistake.
we can't build webrtc successful with tizen 5.5, because tizen 5.5 not support MJPG format.
```
../../libwebrtc/src/tizen/video_capture/help_functions.cc:47:10: error: use of undeclared identifier 'CAMERA_PIXEL_FORMAT_MJPEG'; did you mean 'CAMERA_PIXEL_FORMAT_JPEG'?
    case CAMERA_PIXEL_FORMAT_MJPEG:
         ^~~~~~~~~~~~~~~~~~~~~~~~~
         CAMERA_PIXEL_FORMAT_JPEG
../../webrtc_tizen_tools/sysroot/arm/usr/include/media/camera.h:121:2: note: 'CAMERA_PIXEL_FORMAT_JPEG' declared here
        CAMERA_PIXEL_FORMAT_JPEG,           /**< Encoded pixel format */
        ^
../../libwebrtc/src/tizen/video_capture/help_functions.cc:71:14: error: use of undeclared identifier 'CAMERA_PIXEL_FORMAT_MJPEG'; did you mean 'CAMERA_PIXEL_FORMAT_JPEG'?
      return CAMERA_PIXEL_FORMAT_MJPEG;
             ^~~~~~~~~~~~~~~~~~~~~~~~~
             CAMERA_PIXEL_FORMAT_JPEG

```